website/docs/commands/init.html.markdown

```diff
@@ -15,56 +15,62 @@ from version control. It is safe to run this command multiple times.
 
 ## Usage
 
-Usage: `terraform init [options] [SOURCE] [PATH]`
+Usage: terraform init [options] [DIR]
 
-Initialize a new or existing Terraform working directory by creating
-initial files, loading any remote state, downloading modules, etc.
+  Initialize a new or existing Terraform working directory by creating
+  initial files, loading any remote state, downloading modules, etc.
 
-This is the first command that should be run for any new or existing
-Terraform configuration per machine. This sets up all the local data
-necessary to run Terraform that is typically not committed to version
-control.
+  This is the first command that should be run for any new or existing
+  Terraform configuration per machine. This sets up all the local data
+  necessary to run Terraform that is typically not committed to version
+  control.
 
-This command is always safe to run multiple times. Though subsequent runs
-may give errors, this command will never delete your configuration or state.
-Even so, if you have important information, please back it up prior to
-running this command just in case.
+  This command is always safe to run multiple times. Though subsequent runs
+  may give errors, this command will never delete your configuration or
+  state. Even so, if you have important information, please back it up prior
+  to running this command, just in case.
 
-If no arguments are given, the configuration in the current working directory
-is initialized.
-
-If one or two arguments are given, the first is a SOURCE of a module to
-download to the second argument PATH. After downloading the module to PATH,
-the configuration will be initialized as if this command were called pointing
-only to that PATH. PATH must be empty of any Terraform files. Any
-conflicting non-Terraform files will be overwritten. The module download
-is a copy. If you're downloading a module from Git, it will not preserve
-Git history.
+  If no arguments are given, the configuration in this working directory
+  is initialized.
```

Comment:
Is the whitespace at the front of these lines intentional? Not sure if this is true of the markdown processor we use, but I'd expect this to get interpreted as a preformatted text block with these here, which I assume isn't what we want...